login-style.css

```
@@ -0,0 +1,174 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --gray900: #111827;
+  --gray800: #1f2937;
+  --gray700: #374151;
+  --gray600: #4b5563;
+  --gray500: #6b7280;
+  --gray400: #9ca3af;
+  --gray200: #e5e7eb;
+  --gray100: #f3f4f6;
+  --gray50: #f9fafb;
+  --blue: #3692ff;
+}
+
+html {
+  font-size: 10px;
+  font-family: pretendard, sans-serif;
```

❗️ 수정요청
html 파일에서 걸어주신 [pretendard 링크](https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css)를 보면 font-face에 font명을 "Pretendard"로 정의해두었습니다. 따라서 아래처럼 정의해주세요.

```suggestion
  font-family: Pretendard, sans-serif;
```

더 자세한 내용이 궁금하시면 관련 [github](https://github.com/orioncactus/pretendard)를 확인해보세요!